### DISCUSSION TITLE
Telegram or Discord community

### DISCUSSION THREAD

**Discussion by Chappie18** [charlesbel/Microsoft-Rewards-Farmer#289] [Ideas]
It would be ideal to establish a Telegram or Discord community for connecting with users and assisting them with problems.

**Comment by charlesbel** [comment]
If someone can go ahead and do it, and/or some other persons to do moderation and answers to questions because I'm a bit overloaded by what to do haha (add me in the group as an admin so I would be able to  answer myself too if required)

**Comment by charlesbel** [comment]
And if someone can add every contributors/active people of the repo to this discussion so you can all coordonate

**Reply by Chappie18** [reply]
I've created a Telegram group, and you can join by clicking the link below. I'll make you the administrator.
https://t.me/MSFarmerByCharlesBel

**Reply by Chappie18** [reply]
Could you perhaps include the joining link on the main page so that users can join?
